include/device.h

```diff
@@ -156,6 +156,22 @@ typedef int16_t device_handle_t;
 			pm_control_fn,					\
 			data_ptr, cfg_ptr, level, prio, api_ptr)
 
+/**
+ * @def DEVICE_DT_NAME
+ *
+ * @brief Return a string name for a devicetree node.
+ *
+ * @details This macro returns a string literal name for a devicetree
+ * node.  If a "label" property exists for the give devicetree node
+ * the string value of the "label" property will be returned as the
+ * name.  If there is no "label" property than the node-name@unit-address
+ * for the node will be returned.
```

Comment:
```suggestion
 * @details This macro returns a string literal usable as a device name
 * from a devicetree node. If the node has a "label" property, its value is returned.
 * Otherwise, the node's full "node-name@unit-address" name is returned.
```

Reply:
Done.